tests/baselines/reference/classMemberInitializerScoping2(target=es2017,usedefineforclassfields=true).js

```diff
@@ -0,0 +1,20 @@
+//// [classMemberInitializerScoping2.ts]
+const x = 1
+class C {
+    p = x
+    constructor(x: string) { }
+}
+
+
+//// [classMemberInitializerScoping2.js]
+const x = 1;
+class C {
+    constructor(x) {
+        Object.defineProperty(this, "p", {

```

Comment:
this still incorrectly refers to the outer `x` in the scope of the constructor's `x`, so target=es2017,usedefineforclassfields=true needs an error too